src/styles/components/pods-view.less

```diff
@@ -0,0 +1,125 @@
+.pod-instances-table {
+
+  tbody {
+    line-height: 1.8;
+  }
+
+}
+
+.pod-instances-container-id {
+  color: @white;
+  margin: @base-spacing-unit * -1/4 0;
+  padding: @base-spacing-unit * 1/4 @base-spacing-unit * 1/2;
+
+  &.is-expandable {
```

Comment:
It seems like much of this file is the same as this: https://github.com/dcos/dcos-ui/blob/master/src/styles/components/jobs-view.less#L19

Let's maybe abstract the commonalities into one table type, and then only have specificities in each of these files (pods-view and jobs-view)